coordinator/tests/services/translation/request/agent-unattended.spec.ts

```diff
@@ -112,7 +102,7 @@ describe("createAuthorForWithdraw", () => {
       .extension
     )
       .toBe(
-        professionalCodeExpected
+        sdsCodeExpected
```

Comment:
can we be consistent in how we do these expectations:
line 41 - `expect(id).toStrictEqual(new SDSUniqueId(value))`
line 105 - `expect(id.attributes.extension).toBe(value)`